.gitattributes

```diff
@@ -1 +1,2 @@
-*.strings diff
+*.strings text
+*.strings diff
```

Comment:
@NQuinn27 @jnewc @semirp what is this red circle with a dash?

Reply:
It's applying the effect to all files of type string - `*.string`

https://git-scm.com/docs/gitattributes

Reply:
I was asking about the UI here in GitHub, if you open https://github.com/primer-io/primer-sdk-ios/pull/918/files you will see it.

Reply:
<img width="812" alt="Screenshot 2024-06-28 at 14 52 50" src="https://github.com/primer-io/primer-sdk-ios/assets/137198782/52355e7f-ad84-476d-aa29-1e3fd6704806">